robot/web/web_test.go

```diff
@@ -893,12 +926,13 @@ func TestForeignResource(t *testing.T) {
 	listener := testutils.ReserveRandomListener(t)
 	addr = listener.Addr().String()
 	options.Network.Listener = listener
-	options.Debug = true
 	svc = web.New(injectRobot, logger)
 	err = svc.Start(ctx, options)
 	test.That(t, err, test.ShouldBeNil)
 
-	conn, err = rgrpc.Dial(context.Background(), addr, logger, rpc.WithDialDebug())
```

Comment:
Eric added these debug logs in a while ago to get more information from the `TestForeignResource` hang. The hang doesn't happen here though it happens on L946 below when the gizmo attempts a `DoOne` call.